eng/pipelines/ci.yml

```diff
@@ -15,82 +15,86 @@ pr:
+      os: windows
+      image: windows-2022
+      name: Azure-Pipelines-1ESPT-ExDShared
+    sdl:
+      suppression:
+        suppressionFile: $(Build.SourcesDirectory)\guardian\SDL\.gdnsuppress
+    stages:
+    - stage: Stage
+      jobs:
+        - job: Job
+          steps:
+          - task: servicetree-link-build-task@1
+            displayName: 'ServiceTree Integration'
+            inputs:
+              ServiceTreeGateway: 'ServiceTree Gateway'
+              Service: 'e312ec77-9c77-48a7-9837-32fe6d08d460'
+              BuildOutputUsage: 'test'
+          - task: JavaToolInstaller@0
+            displayName: 'Configure Java 11'
+            inputs:
+              versionSpec: '11'
+              jdkArchitectureOption: 'x64'
+              jdkSourceOption: 'PreInstalled'
+          - task: Gradle@2
+            displayName: 'ktlintCheck'
+            inputs:
+              workingDirectory: 'azure-communication-ui'
+              gradleWrapperFile: 'azure-communication-ui/gradlew'
+              gradleOptions: '-Xmx3072m'
+              tasks: 'ktlintCheck'
+          - task: Gradle@2
+            displayName: 'calling compile, lint, test'
+            inputs:
+              workingDirectory: 'azure-communication-ui/calling'
+              gradleWrapperFile: 'azure-communication-ui/gradlew'
+              gradleOptions: '-Xmx3072m'
+              publishJUnitResults: false
+              tasks: ':calling:build --stacktrace'
+          - task: Gradle@2
+            displayName: 'chat compile, lint, test'
+            inputs:
+              workingDirectory: 'azure-communication-ui'
+              gradleWrapperFile: 'azure-communication-ui/gradlew'
+              gradleOptions: '-Xmx3072m'
+              publishJUnitResults: true
+              testResultsFiles: '**/TEST-*.xml'
+              tasks: ':chat:build --stacktrace'
+          - task: Gradle@2
+            displayName: 'build (compile, lint, test)'
+            inputs:
+              workingDirectory: 'azure-communication-ui'
+              gradleWrapperFile: 'azure-communication-ui/gradlew'
+              gradleOptions: '-Xmx3072m'
+              publishJUnitResults: false
+              tasks: 'build --stacktrace'
+          - task: Gradle@2
+            displayName: 'Coverage report for all packages'
+            inputs:
+              workingDirectory: 'azure-communication-ui'
+              gradleWrapperFile: 'azure-communication-ui/gradlew'
+              gradleOptions: '-Xmx3072m'
+              publishJUnitResults: false
+              tasks: ':combinedCoverageReport'
+          - task: PublishCodeCoverageResults@1
+            displayName: 'Calling - Publish code coverage report'
+            inputs:
+              summaryFileLocation: $(System.DefaultWorkingDirectory)/azure-communication-ui/build/reports/coverage/test/debug/report.xml
+              reportDirectory: $(System.DefaultWorkingDirectory)/azure-communication-ui/build/reports/coverage/test/debug/index.html
```

Comment:
Please test if you see test report with index.html. if not please try to test with 

$(System.DefaultWorkingDirectory)/azure-communication-ui/build/reports/coverage/test/debug/*